### PR TITLE
Migrate tests to jest

### LINKED ISSUE #3: Migrate tests to jest

### PR DESCRIPTION
**Pull request by szeck87**
Closes #3.

### CODE REVIEW

**Comment by szeck87** [comment]
I removed lodash, I forgot it's not needed as a dependency for the test folder.  I don't care for coveralls, their UI isn't great.  codeclimate is nice because it does code analysis in addition to code coverage.  I think it needs to be merged into master first to do a "base" build.

File: package.json [hunk 1]
```diff
@@ -26,13 +27,21 @@
...
   "peerDependencies": {
-    "unibeautify": "^0.5.0",
+    "unibeautify": "^0.7.0",
     "eslint": "^4.17.0"
   },
   "devDependencies": {
+    "@types/jest": "^22.1.2",
+    "@types/lodash": "^4.4.71",
     "@types/node": "^8.9.1",
     "ava": "^0.25.0",
```
Comment (Glavin001) [inline package.json]: Remove Ava

File: test/options/commaDangle.spec.ts [hunk 2]
```diff
@@ -1,17 +1,16 @@
-import test from "ava";
 import { newUnibeautify, Beautifier } from "unibeautify";
-import beautifier from "../../dist";
+import beautifier from "../../src";
 
-test.beforeEach(t => {
-  t.context.unibeautify = newUnibeautify();
+beforeEach(() => {
+  const unibeautify = newUnibeautify();
```
Comment (Glavin001) [inline test/options/commaDangle.spec.ts]: This is no longer used on line 13 and below. Delete.

File: src/index.ts [hunk 3]
```diff
@@ -528,10 +528,10 @@ export const beautifier: Beautifier = {
       //   }
       // ]
     },
-    JSX: true,
-    JavaScript: true
+    JavaScript: true,
+    JSX: true
   },
-  beautify(data) {
+  beautify(data: any) {
```
Comment (Glavin001) [inline src/index.ts]: Should be `any`. I think the interface is called `BeautifyData` or something from Unibeautify.

**Comment by Glavin001** [comment]
@szeck87 missed a few things above 😉 . Please fix first, since this PR has already been merged.

**Pull request merged**